modules/ROOT/pages/upgrade-index.adoc

```diff
@@ -1,35 +1,275 @@
 = Upgrade Runtime Fabric
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:page-aliases: upgrade-cluster.adoc, upgrade-components.adoc
 
-Anypoint Runtime Fabric is a package of components that work together. Updates are delivered to these components in the following methods:
+Anypoint Runtime Fabric is a package of components that work together, and updates are delivered in different ways:
 
-* Cluster-level components are delivered as an updated installer package. These components include Docker, Kubernetes, and the Ops Center.
+* Cluster-level components, which include Docker, Kubernetes, and the Ops Center. These components run on virtual machines or bare-metal servers that act as nodes on the cluster.
++
+Updates are delivered as an updated installer package via the *Runtime Fabric downloads* page in Anypoint Runtime Manager.  The installer package is also referred to as the Runtime Fabric appliance.
 
-* Runtime Fabric components are delivered via Anypoint Runtime Manager in Anypoint Platform. These components include the agent used to connect Runtime Fabric to Anypoint Platform. If an updated application or appliance version is available, a link to the downloads page is displayed in the *Upgrade Available* and *Appliance Version* field, as applicable.
+* Runtime Fabric components, which include the agent used to connect Runtime Fabric to Anypoint Platform.
++
+Updates are delivered via the Runtime Fabric page in Anypoint Runtime Manager. If an updated application or appliance version is available, a link to the downloads page is displayed in the *Upgrade Available* and *Appliance Version* fields, as applicable.
```

Comment:
I think this line makes sense, but it's a bit confusing given we have the appliance / installer package in the line above, but yet we're also referring to it here.

It might make more sense to say that upgrades for both components can be found in Runtime Manager, and just specify how they are applied in the bullet points.

Reply:
I've updated this section.